### PR TITLE
Fixed Issue #1314 Formatting done- Section 6.9.3

### PR DESCRIPTION
**Pull request by AayushiVAgrawal**

### CODE REVIEW

File: .idea/data_viz_reader.iml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (rlwy) [inline .idea/data_viz_reader.iml]: What does this file do?

**Review by pragyansharma24 — APPROVED** [review]

File: 06-process.Rmd [hunk 2]
```diff
@@ -529,13 +529,14 @@ Just because visualization is beautiful to look at doesn’t mean it’s effecti
...
 
-4.  **False sense of security**  Graphics are great for conveying simple ideas fast – but sometimes, they are just not enough. Graphics can make users think they are making data driven decisions or think they fully understand the data when in reality they are only seeing a picture but they don’t know the full story.
+3.   **Lack of Guidance:** It is possible that the user who is interpreting the data lacks training. This can have huge impact on the company. At the same time, analysts could provide clients with incorrect or substandard advice. Even systems with Natural Language Query, expect the user to know what they are looking for. This works with simple data but the industry trend is towards big data, data lakes and complex analysis. It’s so complicated you might not even know what you don’t know, to paraphrase an American Defense secretary. The answer is so simple that its easy to miss.
```
Comment (pragyansharma24) [inline 06-process.Rmd]: The answer is so simple that **it is** easy to miss.

**Review by pragyansharma24 — CHANGES_REQUESTED** [review]
Hi Aayushi, 
Please check if this file was committed knowingly or by mistake.

**Comment by mschermann** [comment]
@AayushiVAgrawal What is the status here?

**Comment by mschermann** [comment]
@AayushiVAgrawal Why is this still open? Please fix and close.

**Comment by AayushiVAgrawal** [comment]
Closed issue #1314

**Pull request merged**